README.md

```diff
@@ -63,7 +64,6 @@ android {
   override fun onCreate() {
         super.onCreate()
         Barricade.Builder(this, BarricadeConfig.getInstance())
-            .enableShakeToStart(this)
```

Comment:
We have this feature available in the latest version `(i.e. v0.0.2)`, therefore we don't need to remove it.

Reply:
added this back